nexus/src/app/instance.rs

```diff
@@ -1321,19 +1321,22 @@ impl super::Nexus {
 
     /// Invoked by a sled agent to publish an updated runtime state for an
     /// Instance.
-    pub(crate) async fn notify_instance_updated(
+    pub(crate) async fn notify_vmm_updated(
```

Comment:
To be honest, I really should probably have renamed this back in #5749, but I was too lazy to. This is much nicer.